libbeat/outputs/elasticsearch/config_otel_test.go

```diff
@@ -20,69 +20,186 @@ package elasticsearch
 import (
 	_ "embed"
 	"fmt"
-	"os"
-	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"go.opentelemetry.io/collector/confmap/confmaptest"
+	"go.opentelemetry.io/collector/confmap"
 	"gopkg.in/yaml.v2"
 
 	"github.com/elastic/elastic-agent-libs/config"
 )
 
 func TestToOtelConfig(t *testing.T) {
 
-	tests := []struct {
-		name      string
-		input     string
-		expOutput string
-		experr    bool
-	}{
-		{
-			name:      "basic elasticsearch input",
-			input:     "basic.yml",
-			expOutput: "basic-op.yml",
-			experr:    false,
-		},
-	}
-
-	for _, test := range tests {
-		t.Run(test.name, func(t *testing.T) {
-			rawConf, err := loadBeatConf(filepath.Join("testdata", test.input))
-			require.NoError(t, err)
-			beatCfg := config.MustNewConfigFrom(rawConf)
-
-			otelCfg, err := ToOTelConfig(beatCfg)
-			require.NoError(t, err, "could not convert beat config to otel ES config")
-
-			expectedValue, err := confmaptest.LoadConf(filepath.Join("testdata", test.expOutput))
-			require.NoError(t, err)
-			want, err := yaml.Marshal(expectedValue.ToStringMap())
-			require.NoError(t, err)
-
-			got, err := yaml.Marshal(otelCfg)
-			require.NoError(t, err)
-
-			assert.Equal(t, string(want), string(got))
-
-		})
-	}
+	t.Run("basic config translation", func(t *testing.T) {
+		basicInput := `
```

Comment:
That's an output configuration, I'm a bit confused why it's called `basicInput`. Maybe it can be called `esOutputCfg`.